frontend/src/metabase-types/types/Metric.ts

```diff
@@ -1,6 +1,6 @@
 import { TableId } from "./Table";
 
-export type MetricId = number;
+export type MetricId = number | string;
```

Comment:
Could this be moved to `metabase-types/api` as well? There are similar ID types for databases and collections.

Reply:
Good idea -- moved the type definition to `metabase-types/api`